src/views/Home.jsx

```diff
@@ -1,8 +1,38 @@
 import './Home.css';
 import { SingleList } from '../components';
+import { useState } from 'react';
+import { createList, useAuth } from '../api';
+import { useNavigate } from 'react-router-dom';
 
 export function Home({ data, setListPath }) {
 	const hasList = data.length !== 0;
+
+	const { user } = useAuth();
+	const userId = user?.uid;
+	const userEmail = user?.email;
+
+	const [inputValue, setInputValue] = useState('');
+	const navigate = useNavigate();
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+
+		try {
+			await createList(userId, userEmail, inputValue);
+			const path = `${user.uid}/${inputValue}`;
+			setListPath(path);
+			setInputValue('');
+			alert('Success: Your New List is Created!');
```

Comment:
Quoting my comment [from PR #22 ](https://github.com/the-collab-lab/tcl-77-smart-shopping-list/pull/22#discussion_r1728006118)

> Something that is nearly universally enforced by a linter, but doesn't appear to be here, is consistency between single and double quotes. I notice some inconsistencies here, which isn't a big deal, but it's usually nice to see picking between one or the other (I'm team double quotes personally) and sticking to it

Reply:
Great observation! I try using single quotes in everything, but some of them get changed to doubles quotes automatically when I save the file. I believe it might be the code formatter I'm using (Prettier) that's causing that. 🤔

Reply:
Ah you're correct! We had a prettier rule enforcing the inconsistency 🤦 I've fixed that [here](https://github.com/the-collab-lab/tcl-77-smart-shopping-list/pull/25)!